src/commands.cpp

```diff
@@ -3394,6 +3426,14 @@ void CommandGetUserData::procresult()
             client->nsr_enabled = nsre;
             client->aplvp_enabled = aplvp;
             client->k = k;
+
+            if (b)
```

Comment:
I think not possible now, but what if a business account becomes a non-business in runtime?
We need to review this code.

Reply:
done!